Assets/Scripts/Score/ScoreDisplayManager.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using TMPro;
+
+public class ScoreDisplayManager : MonoBehaviour
+{
+    [SerializeField]
+    private TextMeshProUGUI highScore,
+                            buttonTxt;
+    private char level;
```

Comment:
I think level should just be an int that is set in the Inspector rather than being calculated from the button text. The current implementation relies on specific button UI (even if that UI isn't likely to change).